src/recordlinker/models/algorithm.py

```diff
@@ -12,9 +12,9 @@ class Algorithm(Base):
     id: orm.Mapped[int] = orm.mapped_column(primary_key=True)
     is_default: orm.Mapped[bool] = orm.mapped_column(default=False, index=True)
     label: orm.Mapped[str] = orm.mapped_column(sqltypes.String(255), unique=True)
-    description: orm.Mapped[str] = orm.mapped_column(sqltypes.Text())
+    description: orm.Mapped[str] = orm.mapped_column(sqltypes.Text(), nullable=True)
```

Comment:
We might want to run an `alembic revision --autogenerate` since changes were made to the `algorithm` model

Reply:
Yeah thats a fair point, I was lax on the alembic changes because of [this upcoming work](https://github.com/CDCgov/RecordLinker/issues/57).  I haven't been happy with it so far, and its honestly a bit early in the project for a tool like this, so I think we can remove for the time being.